### PR TITLE
feat: autoscaling support notification

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:
autoscaling support notification.
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/as' TESTARGS='-run TestAccAsNotification_basic'
...
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/as -v -run TestAccAsNotification_basic -timeout 360m -parallel 4 
=== RUN   TestAccAsNotification_basic 
=== PAUSE TestAccAsNotification_basic
=== CONT  TestAccAsNotification_basic
--- PASS: TestAccAsNotification_basic (130.65s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/as        130.720s
```

### CODE REVIEW

File: docs/resources/as_notification.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+```hcl
+variable "group_id" {}
+variable "topic_urn" {}
+variable "topic_scene" {}
+
+resource "huaweicloud_as_notification" "test" {
+  group_id    = var.group_id
+  topic_urn   = var.topic_urn
+  topic_scene = var.topic_scene
```
Comment (ShiChangkuo) [inline docs/resources/as_notification.md]: the example can not work, please check it

File: huaweicloud/services/as/resource_huaweicloud_as_notification.go [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+		},
+	}
+	putASNotificationOpt.JSONBody = buildCreateOrUpdateASNotificationBodyParams(d)
+	_, err = putASNotificationClient.Request("PUT", putASNotificationPath, &putASNotificationOpt)
+	if err != nil {
+		return diag.Errorf("error creating or updating AS notification: %s", err)
+	}
+
+	topicUrn := d.Get("topic_urn").(string)
+	d.SetId(topicUrn)
```
Comment (hmbig2) [inline huaweicloud/services/as/resource_huaweicloud_as_notification.go:107]: can the user associate a different as_group to the same topic Urn?

Comment (deer-hang) [reply huaweicloud/services/as/resource_huaweicloud_as_notification.go:107]: I have tried it, different as_groups can use the same topic Urn.

**Pull request merged**